torchtune/utils/_import_guard.py

```diff
@@ -20,7 +20,7 @@
 _USE_NEW_TENSOR_CORE_TILED_LAYOUT_API = _is_fbcode() or (
     not _is_fbcode()
     and (
-        ("dev" not in torchao_version and torchao_version >= "0.6.0")
+        ("dev" not in torchao_version and torchao_version >= "0.7.0")
```

Comment:
So the new API is not in 0.6.1? I thought it landed before that release

Reply:
Yeah it's not in 0.6.x